settings.gradle

```diff
@@ -0,0 +1 @@
+rootProject.name = 'apted'
```

Comment:
The file has not changed. This should not be part of the commit.

Reply:
This file is new. It is the proper way to name a gradle project. What do you mean by 'it has not changed'?

Reply:
Again my mistake. The documentation has changed heavily since I read it. This file is fine.